pubspec.yaml

```diff
@@ -155,3 +155,5 @@ flutter:
   #
   # For details regarding fonts from package dependencies,
   # see https://flutter.dev/custom-fonts/#from-packages
+flutter_intl:
+  enabled: true
```

Comment:
Don't you need to install intl_utils in order to use this ?

Reply:
@imGok : this is [IDE's plugin](https://plugins.jetbrains.com/plugin/13666-flutter-intl) which supports to generate boilerplate code for Intl. Have a talk a look at this [Notice](https://pub.dev/packages/intl_utils#usage) also :)

Reply:
Oh sure ! Thanks for docs